lib/system/environ/validate.go

```diff
@@ -56,6 +72,28 @@ func ValidateInstall(env *localenv.LocalEnvironment) func() error {
 	}
 }
 
+// GetServiceName returns the name of the service configured in the specified state directory stateDir
+func GetServiceName(stateDir string) (name string, err error) {
```

Comment:
Nit: Goes to my previous comment about "random" methods in packages - is there a better place for these two (GetServiceName/GetServicePath)? They must be used elsewhere too since they're exported?

Reply:
Why is it a random method? It is in the environment package which implements support for working with the environment, including detecting services based on the contents of the state directory.

Reply:
Okay, that makes sense.